### PR TITLE
Comment if unit tests were not updated. 

### PR DESCRIPTION
**Pull request by OmarShehata**
Posts a quick bullet reminder as described here https://github.com/AnalyticalGraphicsInc/cesium-concierge/issues/92. 

I wasn't sure whether to check for files matching `/.*spec.*/i` vs check if any file under a given directory. I went with the latter, which felt like it would be more general/less chance for an accidental match.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata! 
* :heavy_check_mark: Signed CLA found. 
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium-concierge/blob/unit-test-check/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

---

:sparkles: :sparkles: :sparkles:__I'm the Cesium Concierge and I'm here to test things.__ :sparkles: :sparkles: :sparkles:

File: lib/commentOnOpenedPullRequest.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ commentOnOpenedPullRequest._implementation = function (pullRequestFilesUrl, pull
 
             var askAboutChanges = commentOnOpenedPullRequest._askAboutChanges(files, baseBranch);
             var askAboutThirdParty = commentOnOpenedPullRequest._askAboutThirdParty(files, repositorySettings.thirdPartyFolders);
+            var askAboutTests = commentOnOpenedPullRequest._askAboutTests(files, repositorySettings);
```
Comment (ggetz) [inline lib/commentOnOpenedPullRequest.js]: Why aren't we only passing `repositorySettings.unitTestPath`, like with `askAboutThirdParty`?

File: lib/commentOnOpenedPullRequest.js [hunk 2]
```diff
@@ -136,6 +138,20 @@ commentOnOpenedPullRequest._askAboutThirdParty = function (files, thirdPartyFold
     return false;
 };
 
+commentOnOpenedPullRequest._askAboutTests = function (files, repositorySettings) {
+    if (!defined(repositorySettings.unitTestPath)) {
+        return false;
+    }
+    for (var i = 0; i < files.length; i++) {
+        var regex = new RegExp('^' + repositorySettings.unitTestPath, 'i');
+        console.log(regex.test(files[i]),files[i],repositorySettings.unitTestPath);
```
Comment (ggetz) [inline lib/commentOnOpenedPullRequest.js]: Remove `console.log`

File: lib/templates/pullRequestOpened.hbs [hunk 3]
```diff
@@ -32,3 +32,7 @@ Thanks for the pull request @{{ userName }}!
 * :grey_question: Changes to third party files were made.
     * Looks like a file in one of our ThirdParty folders (`{{ thirdPartyFolders }}`) has been added or modified. Please verify that it has a section in [LICENSE.md]({{ repository_url }}/blob/master/LICENSE.md) and that its license information is up to date with this new version.
 {{/if}}
+{{#if askAboutTests}}
+* :grey_question: Unit tests were not updated.
+    * Make sure you've updated tests to reflect your changes, added tests for any new code and ran the code coverage tool.
```
Comment (ggetz) [inline lib/templates/pullRequestOpened.hbs]: Missing a comma:

"Make sure you've updated tests to reflect your changes, added tests for any new code, and ran the code coverage tool."

File: lib/commentOnOpenedPullRequest.js (hunk 2 above)
Comment (OmarShehata) [reply lib/commentOnOpenedPullRequest.js]: Whoops!

**Comment by OmarShehata** [comment]
Should be good now! I added the link to the testing guide as a hyperlink in the text so it's not too in-your-face if you've already seen it before.

**Comment by ggetz** [comment]
Looks good, thanks @OmarShehata !

We should probably update the template in https://github.com/AnalyticalGraphicsInc/cesium/pull/6757 to reflect this change as well.

**Pull request merged**